01_prepare_host.sh

```diff
@@ -92,6 +92,12 @@ function init_minikube() {
     #If the vm exists, it has already been initialized
     if [[ "$(sudo virsh list --all)" != *"minikube"* ]]; then
       sudo su -l -c "minikube start" "$USER"
+      # Pre-pull the image to reduce pod initialization time
+      for IMAGE_VAR in IRONIC_IMAGE IPA_DOWNLOADER_IMAGE VBMC_IMAGE SUSHY_TOOLS_IMAGE BAREMETAL_OPERATOR_IMAGE; do
+        IMAGE=${!IMAGE_VAR}
+        sudo su -l -c "minikube ssh sudo docker pull "$IMAGE"" "${USER}"
```

Comment:
you do not need the indirect expansion if you use the variables content here directly

Reply:
Copying the same coding conventions used above